### PR TITLE
AT_019.01_JavaScript | Entity > Calendar: Delete record with corrected Xpaths

### PR DESCRIPTION
**Pull request by dashasandler**

### CODE REVIEW

**Review by MariaMoroz — APPROVED** [review]
LGTM

**Review by gmirzoyan — APPROVED** [review]
Good job! All tests were passed.

**Review by TupakAmaru13 — APPROVED** [review]

**Comment by TupakAmaru13** [comment]
Good job!

File: test/specs/CalendarDelrecord.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        const dropDownMenu = "//button[@class='btn btn-round btn-sm btn-primary dropdown-toggle']";
+        $(dropDownMenu).click();
+        const user = "//a[normalize-space()='delete']";
+        $(user).click();
+    });
+    it('12/LP-Should detect if deleted element not exist', () => {
+        const elem = '//a[normalize-space()=\'This is a string\']';
+        const w = $(elem).isExisting();
+    });
+    });
```
Comment (SamJulia) [inline test/specs/CalendarDelrecord.js:82]: very good 👌🏻 next time try to add assertion to check you steps to each it.

**Review by SamJulia — CHANGES_REQUESTED** [review]

**Pull request closed without merging**